app/assets/sass/styles/_code-highlight.scss

```
@@ -0,0 +1,49 @@
+@use "../vendor/nhsuk-frontend" as *;
+@use "../settings" as *;
+
+// Highlight code within body copy
+li code,
+p code,
+td code {
+  font-family: $app-code-font;
+  background-color: nhsuk-colour("white");
+  // stylelint-disable-next-line color-no-hex
+  color: #dd1144;
+  padding: 2px nhsuk-spacing(2);
+  word-break: break-word;
+  @include nhsuk-font-size(16);
+}
+
+// Highlight code in a block
+.app-pre {
```

@frankieroberto Up to you, but every `<pre>` needs to be `<pre class="app-pre">`

Otherwise we can update this selector to be generic:

```suggestion
pre {
```